test/e2e/persistent_volumes.go

```diff
@@ -738,6 +739,109 @@ var _ = framework.KubeDescribe("PersistentVolumes", func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	///////////////////////////////////////////////////////////////////////
+	//				Vsphere Disk
+	///////////////////////////////////////////////////////////////////////
+	// Testing configurations of single a PV/PVC pair attached to a Vsphere Disk
+
+	framework.KubeDescribe("PersistentVolumes:vsphere", func() {
+
+		var (
+			volumePath string
+			pv        *v1.PersistentVolume
+			pvc       *v1.PersistentVolumeClaim
+			clientPod *v1.Pod
+			pvConfig  persistentVolumeConfig
+		)
+		BeforeEach(func() {
+			framework.SkipUnlessProviderIs("vsphere")
+			if volumePath == "" {
+				vsp, err := vsphere.GetVSphere()
+				Expect(err).NotTo(HaveOccurred())
+				var volumeoptions vsphere.VolumeOptions
+				volumeoptions.CapacityKB=1 * 1024 * 1024
+				volumeoptions.Name="e2e-disk" + time.Now().Format("20060102150405")
+				volumeoptions.DiskFormat="thin"
```

Comment:
This test does not verify disk format.  Actually let me do not set DiskFormat in the volumeoptions, as default diskformat is thin.

Objective of this test is to verify Deletion of POD should not fail after associated PV or PVC is deleted. 

We can add other test for disk format verification.